addons/payment/models/res_company.py

```diff
@@ -47,7 +47,7 @@ def _run_payment_onboarding_step(self, menu_id):
         if not stripe_provider:
             base_provider = self.env.ref('payment.payment_provider_stripe')
             # Use sudo to access payment provider record that can be in different company.
-            stripe_provider = base_provider.sudo().copy(default={'company_id': self.env.company.id})
+            stripe_provider = base_provider.sudo().copy(default={'company_id': self.env.company.id, 'website_id': False})
```

Comment:
LGTM! Could you just reformat the line so that it's readable on a split screen? (<= 100 chars/line)

```suggestion
            stripe_provider = base_provider.sudo().copy(default={
                'company_id': self.env.company.id,
                'website_id': False,
            })
```

Reply:
Done!